Sources/SwiftDocC/Model/Rendering/Content/RenderBlockContent.swift

```diff
@@ -124,12 +124,26 @@ public enum RenderBlockContent: Equatable {
         public var code: [String]
         /// Additional metadata for this code block.
         public var metadata: RenderContentMetadata?
+        public var copyToClipboard: Bool = true
+
+        public enum OptionName: String, CaseIterable {
+            case nocopy
+
+            init?<S: StringProtocol>(caseInsensitive raw: S) {
+                self.init(rawValue: raw.lowercased())
+            }
+        }
+
+        public static var knownOptions: Set<String> {
+            Set(OptionName.allCases.map(\.rawValue))
+        }
 
         /// Make a new `CodeListing` with the given data.
-        public init(syntax: String?, code: [String], metadata: RenderContentMetadata?) {
+        public init(syntax: String?, code: [String], metadata: RenderContentMetadata?, copyToClipboard: Bool) {
```

Comment:
This is a source breaking change as anyone who used to call the previous initializer will now have an error that they're missing a parameter.

We either need to deprecate this initializer and keep it for one release and introduce a second initializer with the new parameter (as described in "Introducing source breaking changes" section in CONTIBUTING.md) or we need to provide a default value for this initializer so that existing callers continue to work. 

The second solution is API preserving but not ABI preserving, but that's fine for DocC.